recipe/meta.yaml

```
@@ -1,15 +1,13 @@
 {% set name = "libgit2" %}
-{% set version = "1.0.0" %}
+{% set version = "1.0.1" %}
 
 package:
   name: {{ name|lower }}
   version: {{ version }}
 
 source:
   url: https://github.com/{{ name }}/{{ name }}/archive/v{{ version }}.tar.gz
-  sha256: 6a1fa16a7f6335ce8b2630fbdbb5e57c4027929ebc56fcd1ac55edb141b409b4
-  patches:
-    - htonll-osx.patch
```

```suggestion
    - htonll-osx.patch
```
This patch is still required.

---

ok, I'll rebase it.